docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Organize Data Sources with Source Applications"
+sidebar_position: 1
+sidebar_label: "🆕 Source Applications"
+---
+
+For data collection, you will often have different sources of information that correspond to applications designed for a particular purpose. That is what we will be referring to as Source Applications.
```

Comment:
slightly revised version WDYT?:

```
For data collection, you will often encounter different sources of information that correspond to applications designed for specific purposes. These are what we will refer to as Source Applications.

To illustrate, let's consider Snowplow. We can identify several applications designed for distinct purposes, each serving as a separate data source for behavioral data—or, in other words, a Source Application:

- The Snowplow website, corresponding to the application served at www.snowplow.io.
- The BDP Console application, served at console.snowplowanalytics.com.
- The documentation website, which serves as our information hub for all things related to our product, available at docs.snowplow.io.

Source Applications are a foundational component that enables you to establish the overarching relationships that connect application IDs and [Application Entities].

``` 
on the right the new version

<img width="2301" alt="Screenshot 2024-08-14 at 9 58 02 AM" src="https://github.com/user-attachments/assets/22169549-85b9-4c48-8125-987aa0ff9f7a">